Formula/git-webui.rb

```diff
@@ -0,0 +1,24 @@
+class GitWebui < Formula
+  desc "Local web based user interface for git repositories"
+  homepage "https://github.com/alberthier/git-webui"
+  url "https://github.com/alberthier/git-webui/archive/v1.2.0.tar.gz"
```

Comment:
Looks like there's been a few new versions since this was last reviewed. Would you mind updating to the latest?